b2b-sample/guardio-insurance-sample-application/components/util/successDialog.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import CheckOutlineIcon from '@rsuite/icons/CheckOutline';
+import React from 'react';
+import { Button, Modal } from 'rsuite';
+
+import styles from '../../styles/util.module.css';
+
+export default function SuccessDialog(props) {
+
+    return (
+        <Modal backdrop="static" role="alertdialog" open={props.open} onClose={props.onClose} size="xs">
+            <Modal.Body>
+                <div className={styles.successDialogDiv}>
+                    <CheckOutlineIcon style={{ color: '#0070f3', fontSize: 36 }} />
+                    <h6>User added Successfully.</h6>
```

Comment:
User is added successfully.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/5ffcaa54b0986e1d52996996832e3832ed3d6c9a